oeps/conventional-commits.rst

```diff
@@ -0,0 +1,77 @@
+####################
```

Comment:
As a rollout process question:

Can we trial this proposed set of commit-types for a period of time and get input from devs (somehow) on what's working and what's not?

That is, can we provide a place for devs to provide ongoing feedback on what they find?

- _"None of these types fit for the type of change in this commit."_
- _"Multiple types correspond to this commit and I don't see the value in separating them into multiple commits."_

This feedback will then give us input on how, if at all, we need to ammend.

Reply:
[suggestion] Would it be useful to have a google form that was open for input regarding this spec at all times?  We could then collect issues over time and update the spec as needed.  This would build a cheap feedback loop and if we can link it from the OEP, it would hopefully be pretty easy to find.